lnwallet/channel.go

```diff
@@ -2558,8 +2560,12 @@ func (lc *LightningChannel) CompleteCooperativeClose(remoteSig []byte) (*wire.Ms
 // DeleteState deletes all state concerning the channel from the underlying
 // database, only leaving a small summary describing metadata of the
 // channel's lifetime.
-func (lc *LightningChannel) DeleteState() error {
-	return lc.channelState.CloseChannel()
+func (lc *LightningChannel) MarkChannelAsFullyClosed() error {
```

Comment:
The name of the method was changed, but the `godoc` comment above wasn't updated.